specifications/api/swagger.yaml

```diff
@@ -329,7 +329,13 @@ x-settings: &settings
   properties:
     pool_metadata_source:
       <<: *poolMetadataSource
-      description: How to fetch pool metadata
+      description: |
+        Select stake pool metadata fetching strategy:
+          - `none` - metadata is not fetched at all,
+          - `direct` - metadata is fetched directly from chain,
```

Comment:
That is incorrect. `direct` means that metadata are fetched from URLs declared by stake-pools in their on-chain registration. Only the URL is on chain, not the metadata.

Reply:
Addressed in cbd5355.